src/fides/api/ops/service/authentication/authentication_strategy_api_key.py

```diff
@@ -59,5 +61,17 @@ def add_authentication(
                     query_param.name,
                     param_val,
                 )
+        if self.body:
+            body_with_api_key: Optional[str] = assign_placeholders(
+                self.body, secrets or {}
+            )
+            api_key_request_body: Dict = json.loads(body_with_api_key or "{}")
+
+            original_request_body: Dict = json.loads(request.body or "{}")
+            original_request_body.update(api_key_request_body)
+
+            request.prepare_body(
+                data=json.dumps(original_request_body), files=None, json=True
+            )
```

Comment:
I added this to allow the api authentication strategy to update a request body with an API key.  I think we're defining json as request bodies in our saas connector work but other `body` formats could cause issues.